CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 Breaking changes:
 
+- [#1645](https://github.com/rails-api/active_model_serializers/pull/1645) Expanded KeyTransform into Transform. Changed :dashed transform to :dash. (@remear)
```

Comment:
Should this be named after the manner of what it does, or after its intended use? I originally intended the former, where the class is only responsible for transforming the supplied value and otherwise is not concerned with the transformed value's intended use. It used to transform _only keys of hashes_, so I named it `KeyTransform`. I renamed it to `Transform` since it no longer has such a restriction. A `String` could be passed in for transformation. Everyone, please let me know your thoughts. I'd like to wrap this up shortly.